### PR TITLE
Small refactor on sessions controller

### PR DESCRIPTION
**Pull request by alexventuraio**
To stop from having multiple nested `if` statements.

### CODE REVIEW

File: app/controllers/sessions_controller.rb [hunk 1]
```diff
@@ -3,19 +3,16 @@ class SessionsController < ApplicationController
...
-        redirect_to new_confirmation_path, alert: "You must confirm your email before you can sign in."
-      elsif @user.authenticate(params[:user][:password])
-        login @user
-        redirect_to root_path, notice: "Signed in."
-      else
-        flash[:alert] = "Incorrect email or password."
-        render :new        
-      end
+    
+    render_new_with_alert unless @user
```
Comment (alexventuraio) [inline app/controllers/sessions_controller.rb:7]: should we need to prepend a `return` statement here? 🤔

**Comment by stevepolitodesign** [comment]
@alexventuraio thank you for taking the time to make this PR and #11. I'm still wrapping up this guide, so there's a chance the PR you issued might need to change. I'll ping you here again when it's ready 👍

**Comment by alexventuraio** [comment]
> @alexventuraio thank you for taking the time to make this PR and #11. I'm still wrapping up this guide, so there's a chance the PR you issued might need to change. I'll ping you here again when it's ready 👍

Hi @stevepolitodesign sure thing, thanks for letting me know!

File: app/controllers/sessions_controller.rb [hunk 2]
```diff
@@ -26,5 +23,11 @@ def destroy
...
   end
-
-end
\ No newline at end of file
+  
+  private
+  
+  def render_new_with_alert
+    flash[:alert] = "Incorrect email or password."
+    render :new
```
Comment (tbotes) [inline app/controllers/sessions_controller.rb:31]: Since it's POST, shouldn't this be `redirect_to :new`?

**Comment by stevepolitodesign** [comment]
@alexventuraio I'm going to close this due to conflicts introduced since this PR was issued.

**Pull request closed without merging**